assets/stylesheets/_mixins.scss

```diff
@@ -340,6 +341,7 @@
 @mixin reduce-motion {
 	@media (prefers-reduced-motion: reduce) {
 		animation-duration: 1ms !important;
+		transition-duration: 0s !important;
```

Comment:
Why is this 01 when the animation duration is 1ms? Should they both be 1ms?

Reply:
In the original PR, I noticed some Safari issues when `animation-duration` was set to `0s`: I couldn't get it to reliably render the end-state, which was preventing menus and panels from opening. 😕

I haven't noticed the same issues with `transition` for some reason, so I think it's fine to use `0s` there.